tests/components/ecobee/test_notify.py

```diff
@@ -54,4 +54,4 @@ async def test_legacy_notify_service(
     await hass.async_block_till_done()
     mock_ecobee.send_message.assert_called_with(THERMOSTAT_ID, "It is too cold!")
     mock_ecobee.send_message.reset_mock()
-    assert len(issue_registry.issues) == 1
+    assert len(issue_registry.issues) >= 1
```

Comment:
```suggestion
    assert len(issue_registry.issues) == 2
```
Also below

Reply:
These changes are no longer needed. Have been able to revet these back to the original test assertions.